initramfs/init.sh

```diff
@@ -27,8 +27,13 @@ UPDATE_DEVICES="/dev/mmcblk[0-9]p[0-9]"
 #uc2 : UltiController 2 (UM3,UM3E) This UltiController is not considered here anymore
 #uc3 : UltiController 3 (S5,S5r2,S3)
 DISPLAY_TYPE="uc3"
-UMSPLASH="/umsplash_png.fb"
+UMSPLASH="/SplashUM.fb"
+S3SPLASH="/SplashS3.fb"
```

Comment:
I do not really like the capitals in the file name, but I am fine if you do not want to change it. 
If you would change it, do it in um-u-boot as well.